apps/storybook/src/Table.stories.tsx

```diff
@@ -1344,20 +1402,42 @@ export const ControlledState: Story<Partial<TableProps>> = (args) => {
           />
         ))}
       </InputGroup>
+      <InputGroup label='Control expanded rows' style={{ marginBottom: 11 }}>
+        {data.map((data, index) => (
+          <Checkbox
+            key={index}
+            label={data.name}
+            checked={expandedRows[index]}
+            onChange={(e) => {
+              setExpandedRows((rowIds) => {
+                const expandedRowIds = { ...rowIds };
+                if (e.target.checked) {
+                  expandedRowIds[index] = true;
+                } else {
+                  delete expandedRowIds[index];
+                }
+                return expandedRowIds;
+              });
+            }}
+          />
+        ))}
+      </InputGroup>
       <Table
         columns={columns}
-        data={data}
         emptyTableContent='No data.'
         useControlledState={controlledState}
         stateReducer={tableStateReducer}
         isSelectable
         {...args}
+        data={data}
```

Comment:
This change is to prevent the default `args.data` from overriding the custom `data`